web/modules/custom/dpl_patron_menu/dpl_patron_menu.info.yml

```diff
@@ -0,0 +1,14 @@
+name: DPL menu
+description: Handle the patron user menu
+package: DPL
+dependencies:
+  - drupal:block
+  - dpl_library_agency:dpl_library_agency
+  - dpl_loans:dpl_loans
+#  - dpl_reservation:dpl_reservation
+#  - dpl_fees:dpl_fees
+#  - dpl_favorites_list:dpl_favorites_list
+#  - dpl_dashboard:dpl_dashboard
```

Comment:
If they are not used I think they should be removed. Do you agree?

Reply:
Added the dependencies as they are required to render menu links based on routes from these modules.